src/EditorFeatures/CSharpTest/Diagnostics/UpgradeProject/UpgradeProjectTests.cs

```diff
@@ -72,6 +72,33 @@ void A()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUpgradeProject)]
+        public async Task UpgradeProjectFromCSharp7ToLatest()
+        {
+            await TestLanguageVersionUpgradedAsync(
+@"
+class Program
+{
+#error version:[|7.1|]
+}",
+                LanguageVersion.Latest,
+                new CSharpParseOptions(LanguageVersion.CSharp7));
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUpgradeProject)]
+        public async Task UpgradeProjectFromCSharp7ToCSharp7_1()
+        {
+            await TestLanguageVersionUpgradedAsync(
+@"
+class Program
+{
+#error [|version:7.1|]
```

Comment:
Sorry, I had misread your comment.
When we do a point release, adding the new version is the first thing we do (as features depend on its existence). I have documented a checklist of what needs to be done to complete that first step. That includes fixing the UpgradeProject fixer. But there is a chicken and egg problem, because no feature will trigger the diagnostic requesting this new version. 
That is the purpose of `#error version:<version>`.
This is not the only way the problem can be solved, but it is easy enough and convenient.